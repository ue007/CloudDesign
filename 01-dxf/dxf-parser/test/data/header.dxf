  0
SECTION
  2
HEADER
  9
$ACADVER
  1
AC1027
  9
$ACADMAINTVER
 70
    55
  9
$DWGCODEPAGE
  3
ANSI_1252
  9
$LASTSAVEDBY
  1
Shane
  9
$REQUIREDVERSIONS
160
                 0
  9
$INSBASE
 10
0.0
 20
0.0
 30
0.0
  9
$EXTMIN
 10
146.8403717847312
 20
141.8294137695975
 30
-5.934454319554902
  9
$EXTMAX
 10
762.6679804989191
 20
545.821967323924
 30
6.065545680445099
  9
$LIMMIN
 10
0.0
 20
0.0
  9
$LIMMAX
 10
1000.0
 20
800.0
  9
$ORTHOMODE
 70
     1
  9
$REGENMODE
 70
     1
  9
$FILLMODE
 70
     1
  9
$QTEXTMODE
 70
     0
  9
$MIRRTEXT
 70
     0
  9
$LTSCALE
 40
1.0
  9
$ATTMODE
 70
     1
  9
$TEXTSIZE
 40
0.1875
  9
$TRACEWID
 40
0.05
  9
$TEXTSTYLE
  7
FG-Title
  9
$CLAYER
  8
FG-Dtl-Hidden
  9
$CELTYPE
  6
ByLayer
  9
$CECOLOR
 62
   256
  9
$CELTSCALE
 40
1.0
  9
$DISPSILH
 70
     1
  9
$DIMSCALE
 40
48.0
  9
$DIMASZ
 40
0.125
  9
$DIMEXO
 40
0.0625
  9
$DIMDLI
 40
0.375
  9
$DIMRND
 40
0.0
  9
$DIMDLE
 40
0.0
  9
$DIMEXE
 40
0.0625
  9
$DIMTP
 40
0.0
  9
$DIMTM
 40
0.0
  9
$DIMTXT
 40
0.09375
  9
$DIMCEN
 40
0.09
  9
$DIMTSZ
 40
0.0
  9
$DIMTOL
 70
     0
  9
$DIMLIM
 70
     0
  9
$DIMTIH
 70
     0
  9
$DIMTOH
 70
     0
  9
$DIMSE1
 70
     0
  9
$DIMSE2
 70
     0
  9
$DIMTAD
 70
     1
  9
$DIMZIN
 70
     0
  9
$DIMBLK
  1
Oblique
  9
$DIMASO
 70
     1
  9
$DIMSHO
 70
     1
  9
$DIMPOST
  1
"
  9
$DIMAPOST
  1

  9
$DIMALT
 70
     0
  9
$DIMALTD
 70
     4
  9
$DIMALTF
 40
1.0
  9
$DIMLFAC
 40
1.0
  9
$DIMTOFL
 70
     1
  9
$DIMTVP
 40
0.0
  9
$DIMTIX
 70
     0
  9
$DIMSOXD
 70
     0
  9
$DIMSAH
 70
     0
  9
$DIMBLK1
  1

  9
$DIMBLK2
  1

  9
$DIMSTYLE
  2
STANDARD
  9
$DIMCLRD
 70
   256
  9
$DIMCLRE
 70
   256
  9
$DIMCLRT
 70
   256
  9
$DIMTFAC
 40
1.0
  9
$DIMGAP
 40
0.03125
  9
$DIMJUST
 70
     0
  9
$DIMSD1
 70
     0
  9
$DIMSD2
 70
     0
  9
$DIMTOLJ
 70
     1
  9
$DIMTZIN
 70
     0
  9
$DIMALTZ
 70
     0
  9
$DIMALTTZ
 70
     0
  9
$DIMUPT
 70
     0
  9
$DIMDEC
 70
     4
  9
$DIMTDEC
 70
     4
  9
$DIMALTU
 70
     6
  9
$DIMALTTD
 70
     4
  9
$DIMTXSTY
  7
FG-Note
  9
$DIMAUNIT
 70
     0
  9
$DIMADEC
 70
     0
  9
$DIMALTRND
 40
0.0
  9
$DIMAZIN
 70
     0
  9
$DIMDSEP
 70
    46
  9
$DIMATFIT
 70
     0
  9
$DIMFRAC
 70
     1
  9
$DIMLDRBLK
  1

  9
$DIMLUNIT
 70
     5
  9
$DIMLWD
 70
    -1
  9
$DIMLWE
 70
    -1
  9
$DIMTMOVE
 70
     0
  9
$DIMFXL
 40
0.18
  9
$DIMFXLON
 70
     0
  9
$DIMJOGANG
 40
0.7853981633974483
  9
$DIMTFILL
 70
     0
  9
$DIMTFILLCLR
 70
     0
  9
$DIMARCSYM
 70
     0
  9
$DIMLTYPE
  6
ByLayer
  9
$DIMLTEX1
  6
ByLayer
  9
$DIMLTEX2
  6
ByLayer
  9
$DIMTXTDIRECTION
 70
     0
  9
$LUNITS
 70
     2
  9
$LUPREC
 70
     5
  9
$SKETCHINC
 40
0.1
  9
$FILLETRAD
 40
0.0
  9
$AUNITS
 70
     0
  9
$AUPREC
 70
     0
  9
$MENU
  1
.
  9
$ELEVATION
 40
0.0
  9
$PELEVATION
 40
0.0
  9
$THICKNESS
 40
0.0
  9
$LIMCHECK
 70
     0
  9
$CHAMFERA
 40
0.0
  9
$CHAMFERB
 40
0.0
  9
$CHAMFERC
 40
0.0
  9
$CHAMFERD
 40
0.0
  9
$SKPOLY
 70
     0
  9
$TDCREATE
 40
2455379.770598715
  9
$TDUCREATE
 40
2455379.770598715
  9
$TDUPDATE
 40
2455614.973897338
  9
$TDUUPDATE
 40
2455614.973897338
  9
$TDINDWG
 40
1.6838402431
  9
$TDUSRTIMER
 40
1.6838402431
  9
$USRTIMER
 70
     1
  9
$ANGBASE
 50
0.0
  9
$ANGDIR
 70
     0
  9
$PDMODE
 70
     0
  9
$PDSIZE
 40
0.0
  9
$PLINEWID
 40
0.0
  9
$SPLFRAME
 70
     0
  9
$SPLINETYPE
 70
     6
  9
$SPLINESEGS
 70
     8
  9
$HANDSEED
  5
BFA8
  9
$SURFTAB1
 70
     6
  9
$SURFTAB2
 70
     6
  9
$SURFTYPE
 70
     6
  9
$SURFU
 70
     6
  9
$SURFV
 70
     6
  9
$UCSBASE
  2

  9
$UCSNAME
  2

  9
$UCSORG
 10
0.0
 20
0.0
 30
0.0
  9
$UCSXDIR
 10
1.0
 20
0.0
 30
0.0
  9
$UCSYDIR
 10
0.0
 20
1.0
 30
0.0
  9
$UCSORTHOREF
  2

  9
$UCSORTHOVIEW
 70
     0
  9
$UCSORGTOP
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGBOTTOM
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGLEFT
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGRIGHT
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGFRONT
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGBACK
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSBASE
  2

  9
$PUCSNAME
  2

  9
$PUCSORG
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSXDIR
 10
1.0
 20
0.0
 30
0.0
  9
$PUCSYDIR
 10
0.0
 20
1.0
 30
0.0
  9
$PUCSORTHOREF
  2

  9
$PUCSORTHOVIEW
 70
     0
  9
$PUCSORGTOP
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGBOTTOM
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGLEFT
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGRIGHT
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGFRONT
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGBACK
 10
0.0
 20
0.0
 30
0.0
  9
$USERI1
 70
     0
  9
$USERI2
 70
     0
  9
$USERI3
 70
     0
  9
$USERI4
 70
     0
  9
$USERI5
 70
     0
  9
$USERR1
 40
0.0
  9
$USERR2
 40
0.0
  9
$USERR3
 40
0.0
  9
$USERR4
 40
0.0
  9
$USERR5
 40
0.0
  9
$WORLDVIEW
 70
     1
  9
$SHADEDGE
 70
     3
  9
$SHADEDIF
 70
    70
  9
$TILEMODE
 70
     0
  9
$MAXACTVP
 70
    64
  9
$PINSBASE
 10
0.0
 20
0.0
 30
0.0
  9
$PLIMCHECK
 70
     0
  9
$PEXTMIN
 10
-0.25
 20
-0.1305130401371635
 30
0.0
  9
$PEXTMAX
 10
16.5
 20
10.61948695986283
 30
0.0
  9
$PLIMMIN
 10
-0.2499321702471884
 20
-0.3750450970290743
  9
$PLIMMAX
 10
16.75006734916103
 20
10.62495466267503
  9
$UNITMODE
 70
     0
  9
$VISRETAIN
 70
     1
  9
$PLINEGEN
 70
     0
  9
$PSLTSCALE
 70
     1
  9
$TREEDEPTH
 70
  3020
  9
$CMLSTYLE
  2
Standard
  9
$CMLJUST
 70
     0
  9
$CMLSCALE
 40
1.0
  9
$PROXYGRAPHICS
 70
     1
  9
$MEASUREMENT
 70
     0
  9
$CELWEIGHT
370
    -1
  9
$ENDCAPS
280
     0
  9
$JOINSTYLE
280
     0
  9
$LWDISPLAY
290
     0
  9
$INSUNITS
 70
     0
  9
$HYPERLINKBASE
  1

  9
$STYLESHEET
  1
Florida.ctb
  9
$XEDIT
290
     1
  9
$CEPSNTYPE
380
     0
  9
$PSTYLEMODE
290
     1
  9
$FINGERPRINTGUID
  2
{E451773B-08B1-4376-9DA5-8F5F0D6C19EA}
  9
$VERSIONGUID
  2
{6EFBAB5D-9128-4E51-B2D4-822E03762ACA}
  9
$EXTNAMES
290
     1
  9
$PSVPSCALE
 40
0.0
  9
$OLESTARTUP
290
     0
  9
$SORTENTS
280
   127
  9
$INDEXCTL
280
     0
  9
$HIDETEXT
280
     1
  9
$XCLIPFRAME
280
     0
  9
$HALOGAP
280
     0
  9
$OBSCOLOR
 70
   257
  9
$OBSLTYPE
280
     0
  9
$INTERSECTIONDISPLAY
280
     0
  9
$INTERSECTIONCOLOR
 70
   257
  9
$DIMASSOC
280
     2
  9
$PROJECTNAME
  1

  9
$CAMERADISPLAY
290
     0
  9
$LENSLENGTH
 40
50.0
  9
$CAMERAHEIGHT
 40
0.0
  9
$STEPSPERSEC
 40
2.0
  9
$STEPSIZE
 40
6.0
  9
$3DDWFPREC
 40
2.0
  9
$PSOLWIDTH
 40
0.25
  9
$PSOLHEIGHT
 40
4.0
  9
$LOFTANG1
 40
1.570796326794896
  9
$LOFTANG2
 40
1.570796326794896
  9
$LOFTMAG1
 40
0.0
  9
$LOFTMAG2
 40
0.0
  9
$LOFTPARAM
 70
     7
  9
$LOFTNORMALS
280
     1
  9
$LATITUDE
 40
37.795
  9
$LONGITUDE
 40
-122.394
  9
$NORTHDIRECTION
 40
0.0
  9
$TIMEZONE
 70
 -8000
  9
$LIGHTGLYPHDISPLAY
280
     1
  9
$TILEMODELIGHTSYNCH
280
     1
  9
$CMATERIAL
347
96
  9
$SOLIDHIST
280
     0
  9
$SHOWHIST
280
     1
  9
$DWFFRAME
280
     2
  9
$DGNFRAME
280
     0
  9
$REALWORLDSCALE
290
     1
  9
$INTERFERECOLOR
 62
     1
  9
$INTERFEREOBJVS
345
A3
  9
$INTERFEREVPVS
346
BF84
  9
$CSHADOW
280
     0
  9
$SHADOWPLANELOCATION
 40
0.0
  0
ENDSEC
  0
EOF