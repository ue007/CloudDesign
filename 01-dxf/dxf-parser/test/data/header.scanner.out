[{"code":0,"value":"SECTION"},{"code":2,"value":"HEADER"},{"code":9,"value":"$ACADVER"},{"code":1,"value":"AC1027"},{"code":9,"value":"$ACADMAINTVER"},{"code":70,"value":55},{"code":9,"value":"$DWGCODEPAGE"},{"code":3,"value":"ANSI_1252"},{"code":9,"value":"$LASTSAVEDBY"},{"code":1,"value":"Shane"},{"code":9,"value":"$REQUIREDVERSIONS"},{"code":160,"value":0},{"code":9,"value":"$INSBASE"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$EXTMIN"},{"code":10,"value":146.8403717847312},{"code":20,"value":141.8294137695975},{"code":30,"value":-5.934454319554902},{"code":9,"value":"$EXTMAX"},{"code":10,"value":762.6679804989191},{"code":20,"value":545.821967323924},{"code":30,"value":6.065545680445099},{"code":9,"value":"$LIMMIN"},{"code":10,"value":0},{"code":20,"value":0},{"code":9,"value":"$LIMMAX"},{"code":10,"value":1000},{"code":20,"value":800},{"code":9,"value":"$ORTHOMODE"},{"code":70,"value":1},{"code":9,"value":"$REGENMODE"},{"code":70,"value":1},{"code":9,"value":"$FILLMODE"},{"code":70,"value":1},{"code":9,"value":"$QTEXTMODE"},{"code":70,"value":0},{"code":9,"value":"$MIRRTEXT"},{"code":70,"value":0},{"code":9,"value":"$LTSCALE"},{"code":40,"value":1},{"code":9,"value":"$ATTMODE"},{"code":70,"value":1},{"code":9,"value":"$TEXTSIZE"},{"code":40,"value":0.1875},{"code":9,"value":"$TRACEWID"},{"code":40,"value":0.05},{"code":9,"value":"$TEXTSTYLE"},{"code":7,"value":"FG-Title"},{"code":9,"value":"$CLAYER"},{"code":8,"value":"FG-Dtl-Hidden"},{"code":9,"value":"$CELTYPE"},{"code":6,"value":"ByLayer"},{"code":9,"value":"$CECOLOR"},{"code":62,"value":256},{"code":9,"value":"$CELTSCALE"},{"code":40,"value":1},{"code":9,"value":"$DISPSILH"},{"code":70,"value":1},{"code":9,"value":"$DIMSCALE"},{"code":40,"value":48},{"code":9,"value":"$DIMASZ"},{"code":40,"value":0.125},{"code":9,"value":"$DIMEXO"},{"code":40,"value":0.0625},{"code":9,"value":"$DIMDLI"},{"code":40,"value":0.375},{"code":9,"value":"$DIMRND"},{"code":40,"value":0},{"code":9,"value":"$DIMDLE"},{"code":40,"value":0},{"code":9,"value":"$DIMEXE"},{"code":40,"value":0.0625},{"code":9,"value":"$DIMTP"},{"code":40,"value":0},{"code":9,"value":"$DIMTM"},{"code":40,"value":0},{"code":9,"value":"$DIMTXT"},{"code":40,"value":0.09375},{"code":9,"value":"$DIMCEN"},{"code":40,"value":0.09},{"code":9,"value":"$DIMTSZ"},{"code":40,"value":0},{"code":9,"value":"$DIMTOL"},{"code":70,"value":0},{"code":9,"value":"$DIMLIM"},{"code":70,"value":0},{"code":9,"value":"$DIMTIH"},{"code":70,"value":0},{"code":9,"value":"$DIMTOH"},{"code":70,"value":0},{"code":9,"value":"$DIMSE1"},{"code":70,"value":0},{"code":9,"value":"$DIMSE2"},{"code":70,"value":0},{"code":9,"value":"$DIMTAD"},{"code":70,"value":1},{"code":9,"value":"$DIMZIN"},{"code":70,"value":0},{"code":9,"value":"$DIMBLK"},{"code":1,"value":"Oblique"},{"code":9,"value":"$DIMASO"},{"code":70,"value":1},{"code":9,"value":"$DIMSHO"},{"code":70,"value":1},{"code":9,"value":"$DIMPOST"},{"code":1,"value":"\""},{"code":9,"value":"$DIMAPOST"},{"code":1,"value":""},{"code":9,"value":"$DIMALT"},{"code":70,"value":0},{"code":9,"value":"$DIMALTD"},{"code":70,"value":4},{"code":9,"value":"$DIMALTF"},{"code":40,"value":1},{"code":9,"value":"$DIMLFAC"},{"code":40,"value":1},{"code":9,"value":"$DIMTOFL"},{"code":70,"value":1},{"code":9,"value":"$DIMTVP"},{"code":40,"value":0},{"code":9,"value":"$DIMTIX"},{"code":70,"value":0},{"code":9,"value":"$DIMSOXD"},{"code":70,"value":0},{"code":9,"value":"$DIMSAH"},{"code":70,"value":0},{"code":9,"value":"$DIMBLK1"},{"code":1,"value":""},{"code":9,"value":"$DIMBLK2"},{"code":1,"value":""},{"code":9,"value":"$DIMSTYLE"},{"code":2,"value":"STANDARD"},{"code":9,"value":"$DIMCLRD"},{"code":70,"value":256},{"code":9,"value":"$DIMCLRE"},{"code":70,"value":256},{"code":9,"value":"$DIMCLRT"},{"code":70,"value":256},{"code":9,"value":"$DIMTFAC"},{"code":40,"value":1},{"code":9,"value":"$DIMGAP"},{"code":40,"value":0.03125},{"code":9,"value":"$DIMJUST"},{"code":70,"value":0},{"code":9,"value":"$DIMSD1"},{"code":70,"value":0},{"code":9,"value":"$DIMSD2"},{"code":70,"value":0},{"code":9,"value":"$DIMTOLJ"},{"code":70,"value":1},{"code":9,"value":"$DIMTZIN"},{"code":70,"value":0},{"code":9,"value":"$DIMALTZ"},{"code":70,"value":0},{"code":9,"value":"$DIMALTTZ"},{"code":70,"value":0},{"code":9,"value":"$DIMUPT"},{"code":70,"value":0},{"code":9,"value":"$DIMDEC"},{"code":70,"value":4},{"code":9,"value":"$DIMTDEC"},{"code":70,"value":4},{"code":9,"value":"$DIMALTU"},{"code":70,"value":6},{"code":9,"value":"$DIMALTTD"},{"code":70,"value":4},{"code":9,"value":"$DIMTXSTY"},{"code":7,"value":"FG-Note"},{"code":9,"value":"$DIMAUNIT"},{"code":70,"value":0},{"code":9,"value":"$DIMADEC"},{"code":70,"value":0},{"code":9,"value":"$DIMALTRND"},{"code":40,"value":0},{"code":9,"value":"$DIMAZIN"},{"code":70,"value":0},{"code":9,"value":"$DIMDSEP"},{"code":70,"value":46},{"code":9,"value":"$DIMATFIT"},{"code":70,"value":0},{"code":9,"value":"$DIMFRAC"},{"code":70,"value":1},{"code":9,"value":"$DIMLDRBLK"},{"code":1,"value":""},{"code":9,"value":"$DIMLUNIT"},{"code":70,"value":5},{"code":9,"value":"$DIMLWD"},{"code":70,"value":-1},{"code":9,"value":"$DIMLWE"},{"code":70,"value":-1},{"code":9,"value":"$DIMTMOVE"},{"code":70,"value":0},{"code":9,"value":"$DIMFXL"},{"code":40,"value":0.18},{"code":9,"value":"$DIMFXLON"},{"code":70,"value":0},{"code":9,"value":"$DIMJOGANG"},{"code":40,"value":0.7853981633974483},{"code":9,"value":"$DIMTFILL"},{"code":70,"value":0},{"code":9,"value":"$DIMTFILLCLR"},{"code":70,"value":0},{"code":9,"value":"$DIMARCSYM"},{"code":70,"value":0},{"code":9,"value":"$DIMLTYPE"},{"code":6,"value":"ByLayer"},{"code":9,"value":"$DIMLTEX1"},{"code":6,"value":"ByLayer"},{"code":9,"value":"$DIMLTEX2"},{"code":6,"value":"ByLayer"},{"code":9,"value":"$DIMTXTDIRECTION"},{"code":70,"value":0},{"code":9,"value":"$LUNITS"},{"code":70,"value":2},{"code":9,"value":"$LUPREC"},{"code":70,"value":5},{"code":9,"value":"$SKETCHINC"},{"code":40,"value":0.1},{"code":9,"value":"$FILLETRAD"},{"code":40,"value":0},{"code":9,"value":"$AUNITS"},{"code":70,"value":0},{"code":9,"value":"$AUPREC"},{"code":70,"value":0},{"code":9,"value":"$MENU"},{"code":1,"value":"."},{"code":9,"value":"$ELEVATION"},{"code":40,"value":0},{"code":9,"value":"$PELEVATION"},{"code":40,"value":0},{"code":9,"value":"$THICKNESS"},{"code":40,"value":0},{"code":9,"value":"$LIMCHECK"},{"code":70,"value":0},{"code":9,"value":"$CHAMFERA"},{"code":40,"value":0},{"code":9,"value":"$CHAMFERB"},{"code":40,"value":0},{"code":9,"value":"$CHAMFERC"},{"code":40,"value":0},{"code":9,"value":"$CHAMFERD"},{"code":40,"value":0},{"code":9,"value":"$SKPOLY"},{"code":70,"value":0},{"code":9,"value":"$TDCREATE"},{"code":40,"value":2455379.770598715},{"code":9,"value":"$TDUCREATE"},{"code":40,"value":2455379.770598715},{"code":9,"value":"$TDUPDATE"},{"code":40,"value":2455614.973897338},{"code":9,"value":"$TDUUPDATE"},{"code":40,"value":2455614.973897338},{"code":9,"value":"$TDINDWG"},{"code":40,"value":1.6838402431},{"code":9,"value":"$TDUSRTIMER"},{"code":40,"value":1.6838402431},{"code":9,"value":"$USRTIMER"},{"code":70,"value":1},{"code":9,"value":"$ANGBASE"},{"code":50,"value":0},{"code":9,"value":"$ANGDIR"},{"code":70,"value":0},{"code":9,"value":"$PDMODE"},{"code":70,"value":0},{"code":9,"value":"$PDSIZE"},{"code":40,"value":0},{"code":9,"value":"$PLINEWID"},{"code":40,"value":0},{"code":9,"value":"$SPLFRAME"},{"code":70,"value":0},{"code":9,"value":"$SPLINETYPE"},{"code":70,"value":6},{"code":9,"value":"$SPLINESEGS"},{"code":70,"value":8},{"code":9,"value":"$HANDSEED"},{"code":5,"value":"BFA8"},{"code":9,"value":"$SURFTAB1"},{"code":70,"value":6},{"code":9,"value":"$SURFTAB2"},{"code":70,"value":6},{"code":9,"value":"$SURFTYPE"},{"code":70,"value":6},{"code":9,"value":"$SURFU"},{"code":70,"value":6},{"code":9,"value":"$SURFV"},{"code":70,"value":6},{"code":9,"value":"$UCSBASE"},{"code":2,"value":""},{"code":9,"value":"$UCSNAME"},{"code":2,"value":""},{"code":9,"value":"$UCSORG"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSXDIR"},{"code":10,"value":1},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSYDIR"},{"code":10,"value":0},{"code":20,"value":1},{"code":30,"value":0},{"code":9,"value":"$UCSORTHOREF"},{"code":2,"value":""},{"code":9,"value":"$UCSORTHOVIEW"},{"code":70,"value":0},{"code":9,"value":"$UCSORGTOP"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSORGBOTTOM"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSORGLEFT"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSORGRIGHT"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSORGFRONT"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$UCSORGBACK"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSBASE"},{"code":2,"value":""},{"code":9,"value":"$PUCSNAME"},{"code":2,"value":""},{"code":9,"value":"$PUCSORG"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSXDIR"},{"code":10,"value":1},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSYDIR"},{"code":10,"value":0},{"code":20,"value":1},{"code":30,"value":0},{"code":9,"value":"$PUCSORTHOREF"},{"code":2,"value":""},{"code":9,"value":"$PUCSORTHOVIEW"},{"code":70,"value":0},{"code":9,"value":"$PUCSORGTOP"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSORGBOTTOM"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSORGLEFT"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSORGRIGHT"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSORGFRONT"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PUCSORGBACK"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$USERI1"},{"code":70,"value":0},{"code":9,"value":"$USERI2"},{"code":70,"value":0},{"code":9,"value":"$USERI3"},{"code":70,"value":0},{"code":9,"value":"$USERI4"},{"code":70,"value":0},{"code":9,"value":"$USERI5"},{"code":70,"value":0},{"code":9,"value":"$USERR1"},{"code":40,"value":0},{"code":9,"value":"$USERR2"},{"code":40,"value":0},{"code":9,"value":"$USERR3"},{"code":40,"value":0},{"code":9,"value":"$USERR4"},{"code":40,"value":0},{"code":9,"value":"$USERR5"},{"code":40,"value":0},{"code":9,"value":"$WORLDVIEW"},{"code":70,"value":1},{"code":9,"value":"$SHADEDGE"},{"code":70,"value":3},{"code":9,"value":"$SHADEDIF"},{"code":70,"value":70},{"code":9,"value":"$TILEMODE"},{"code":70,"value":0},{"code":9,"value":"$MAXACTVP"},{"code":70,"value":64},{"code":9,"value":"$PINSBASE"},{"code":10,"value":0},{"code":20,"value":0},{"code":30,"value":0},{"code":9,"value":"$PLIMCHECK"},{"code":70,"value":0},{"code":9,"value":"$PEXTMIN"},{"code":10,"value":-0.25},{"code":20,"value":-0.1305130401371635},{"code":30,"value":0},{"code":9,"value":"$PEXTMAX"},{"code":10,"value":16.5},{"code":20,"value":10.61948695986283},{"code":30,"value":0},{"code":9,"value":"$PLIMMIN"},{"code":10,"value":-0.2499321702471884},{"code":20,"value":-0.3750450970290743},{"code":9,"value":"$PLIMMAX"},{"code":10,"value":16.75006734916103},{"code":20,"value":10.62495466267503},{"code":9,"value":"$UNITMODE"},{"code":70,"value":0},{"code":9,"value":"$VISRETAIN"},{"code":70,"value":1},{"code":9,"value":"$PLINEGEN"},{"code":70,"value":0},{"code":9,"value":"$PSLTSCALE"},{"code":70,"value":1},{"code":9,"value":"$TREEDEPTH"},{"code":70,"value":3020},{"code":9,"value":"$CMLSTYLE"},{"code":2,"value":"Standard"},{"code":9,"value":"$CMLJUST"},{"code":70,"value":0},{"code":9,"value":"$CMLSCALE"},{"code":40,"value":1},{"code":9,"value":"$PROXYGRAPHICS"},{"code":70,"value":1},{"code":9,"value":"$MEASUREMENT"},{"code":70,"value":0},{"code":9,"value":"$CELWEIGHT"},{"code":370,"value":-1},{"code":9,"value":"$ENDCAPS"},{"code":280,"value":0},{"code":9,"value":"$JOINSTYLE"},{"code":280,"value":0},{"code":9,"value":"$LWDISPLAY"},{"code":290,"value":false},{"code":9,"value":"$INSUNITS"},{"code":70,"value":0},{"code":9,"value":"$HYPERLINKBASE"},{"code":1,"value":""},{"code":9,"value":"$STYLESHEET"},{"code":1,"value":"Florida.ctb"},{"code":9,"value":"$XEDIT"},{"code":290,"value":true},{"code":9,"value":"$CEPSNTYPE"},{"code":380,"value":0},{"code":9,"value":"$PSTYLEMODE"},{"code":290,"value":true},{"code":9,"value":"$FINGERPRINTGUID"},{"code":2,"value":"{E451773B-08B1-4376-9DA5-8F5F0D6C19EA}"},{"code":9,"value":"$VERSIONGUID"},{"code":2,"value":"{6EFBAB5D-9128-4E51-B2D4-822E03762ACA}"},{"code":9,"value":"$EXTNAMES"},{"code":290,"value":true},{"code":9,"value":"$PSVPSCALE"},{"code":40,"value":0},{"code":9,"value":"$OLESTARTUP"},{"code":290,"value":false},{"code":9,"value":"$SORTENTS"},{"code":280,"value":127},{"code":9,"value":"$INDEXCTL"},{"code":280,"value":0},{"code":9,"value":"$HIDETEXT"},{"code":280,"value":1},{"code":9,"value":"$XCLIPFRAME"},{"code":280,"value":0},{"code":9,"value":"$HALOGAP"},{"code":280,"value":0},{"code":9,"value":"$OBSCOLOR"},{"code":70,"value":257},{"code":9,"value":"$OBSLTYPE"},{"code":280,"value":0},{"code":9,"value":"$INTERSECTIONDISPLAY"},{"code":280,"value":0},{"code":9,"value":"$INTERSECTIONCOLOR"},{"code":70,"value":257},{"code":9,"value":"$DIMASSOC"},{"code":280,"value":2},{"code":9,"value":"$PROJECTNAME"},{"code":1,"value":""},{"code":9,"value":"$CAMERADISPLAY"},{"code":290,"value":false},{"code":9,"value":"$LENSLENGTH"},{"code":40,"value":50},{"code":9,"value":"$CAMERAHEIGHT"},{"code":40,"value":0},{"code":9,"value":"$STEPSPERSEC"},{"code":40,"value":2},{"code":9,"value":"$STEPSIZE"},{"code":40,"value":6},{"code":9,"value":"$3DDWFPREC"},{"code":40,"value":2},{"code":9,"value":"$PSOLWIDTH"},{"code":40,"value":0.25},{"code":9,"value":"$PSOLHEIGHT"},{"code":40,"value":4},{"code":9,"value":"$LOFTANG1"},{"code":40,"value":1.570796326794896},{"code":9,"value":"$LOFTANG2"},{"code":40,"value":1.570796326794896},{"code":9,"value":"$LOFTMAG1"},{"code":40,"value":0},{"code":9,"value":"$LOFTMAG2"},{"code":40,"value":0},{"code":9,"value":"$LOFTPARAM"},{"code":70,"value":7},{"code":9,"value":"$LOFTNORMALS"},{"code":280,"value":1},{"code":9,"value":"$LATITUDE"},{"code":40,"value":37.795},{"code":9,"value":"$LONGITUDE"},{"code":40,"value":-122.394},{"code":9,"value":"$NORTHDIRECTION"},{"code":40,"value":0},{"code":9,"value":"$TIMEZONE"},{"code":70,"value":-8000},{"code":9,"value":"$LIGHTGLYPHDISPLAY"},{"code":280,"value":1},{"code":9,"value":"$TILEMODELIGHTSYNCH"},{"code":280,"value":1},{"code":9,"value":"$CMATERIAL"},{"code":347,"value":"96"},{"code":9,"value":"$SOLIDHIST"},{"code":280,"value":0},{"code":9,"value":"$SHOWHIST"},{"code":280,"value":1},{"code":9,"value":"$DWFFRAME"},{"code":280,"value":2},{"code":9,"value":"$DGNFRAME"},{"code":280,"value":0},{"code":9,"value":"$REALWORLDSCALE"},{"code":290,"value":true},{"code":9,"value":"$INTERFERECOLOR"},{"code":62,"value":1},{"code":9,"value":"$INTERFEREOBJVS"},{"code":345,"value":"A3"},{"code":9,"value":"$INTERFEREVPVS"},{"code":346,"value":"BF84"},{"code":9,"value":"$CSHADOW"},{"code":280,"value":0},{"code":9,"value":"$SHADOWPLANELOCATION"},{"code":40,"value":0},{"code":0,"value":"ENDSEC"},{"code":0,"value":"EOF"}]